vmdb/app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -184,24 +184,29 @@ def quota_find_provisions(options)
     scheduled_range = self.quota_get_time_range(options[:scheduled_time])
     scheduled_today = scheduled_range.first == self.quota_get_time_range().first
 
-    queued_requests = MiqQueue.find(:all, :conditions => ["class_name = 'MiqProvisionRequest' and method_name = 'create_provision_instances' and state = 'ready' and (deliver_on >= ? and deliver_on < ?)",
-                                *scheduled_range])
+    queued_requests = MiqQueue.where(
+                        :class_name => 'MiqProvisionRequest',
+                        :method_name => 'create_provision_instances',
+                        :state => 'ready',
+                        :deliver_on => scheduled_range
```

Comment:
Done. Also fixed a couple of instances of Good And Proper Hash Style that my fingers mischievously snuck in :grin:.